pipeline/ingestor/ingestor_common/src/test/scala/weco/pipeline/ingestor/models/WorkQueryableValuesTest.scala

```diff
@@ -163,13 +166,150 @@ class WorkQueryableValuesTest
+        workData = WorkData[DataState.Identified](),
+        relations = Relations(),
+        availabilities = Set()
+      )
+      q.genreConceptIds shouldBe Nil
+    }
+
+    it(
+      "produces a list consisting of the canonicalIds for each Genre concept"
+    ) {
+      val data = WorkData[DataState.Identified](
+        title = Some(s"title-${randomAlphanumeric(length = 10)}"),
+        genres = List(
+          Genre(
+            label = "Etching",
+            concepts = List(
+              genreConceptWithCanonicalId(
+                label = "Etching",
+                canonicalId = "qfsrdqed"
+              )
+            )
+          ),
+          Genre(
+            label = "Netsuke",
+            concepts = List(
+              genreConceptWithCanonicalId(
+                label = "Netsuke",
+                canonicalId = "xe5nyufs"
+              )
+            )
+          )
+        )
+      )
+      val q = WorkQueryableValues(
+        id = createCanonicalId,
+        sourceIdentifier = createSourceIdentifier,
+        workData = data,
+        relations = Relations(),
+        availabilities = Set()
+      )
+      q.genreConceptIds shouldBe List("qfsrdqed", "xe5nyufs")
+    }
+
+    it(
+      "Only uses the first concept in each concepts list"
+    ) {
+      info("The first concept is the one pertaining to the actual genre.")
+      info("Other concepts in the list are just extra information.")
+      info("It may be worthwhile exposing them to queries,")
+      info("but the known use-case for this field is to link between genres")
+      info(
+        "rather than (e.g.) cities or periods, which are inconsistently marked up in the source data"
+      )
+
+      val data = WorkData[DataState.Identified](
+        title = Some(s"title-${randomAlphanumeric(length = 10)}"),
+        genres = List(
+          Genre(
+            label = "Etching",
+            concepts = List(
+              genreConceptWithCanonicalId(
+                label = "Etching",
+                canonicalId = "qfsrdqed"
+              ),
+              conceptWithCanonicalId(
+                label = "Base 18th century humour",
+                canonicalId = "cyskv5me"
+              )
+            )
+          ),
+          Genre(
+            label = "Netsuke",
+            concepts = List(
+              genreConceptWithCanonicalId(
+                label = "Netsuke",
+                canonicalId = "xe5nyufs"
+              ),
+              conceptWithCanonicalId(
+                label = "Tanuki",
+                canonicalId = "stqe4kw6"
+              )
+            )
+          )
+        )
+      )
+      val q = WorkQueryableValues(
+        id = createCanonicalId,
+        sourceIdentifier = createSourceIdentifier,
+        workData = data,
+        relations = Relations(),
+        availabilities = Set()
+      )
+      q.genreConceptIds shouldBe List("qfsrdqed", "xe5nyufs")
+      // yxey85un
```

Comment:
What's this stray ID for?

Reply:
whoops, a leftover from an earlier incarnation of this test